packages/core/src/exchanges/ssr.ts

```diff
@@ -35,10 +35,11 @@ const serializeResult = ({
   data,
   error,
 }: OperationResult): SerializedResult => {
-  const result: SerializedResult = {
-    data: JSON.stringify(data),
-    error: undefined,
-  };
+  const result: SerializedResult = {};
+
+  if (data) {
```

Comment:
```suggestion
  if (data !== undefined) {
```

since it can be `null`

Reply:
I'm also wondering whether we should serialise `extensions`?